terraform/environment/.terraform.lock.hcl

```diff
@@ -0,0 +1,60 @@
+# This file is maintained automatically by "terraform init".
+# Manual edits may be lost in future updates.
+
```

Comment:
Should this lock file definitely be committed?  If so , fine. If not, could it be added to .gitignore?

Reply:
commit it for now